composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "docler-labs/codeception-slim-module",
+    "description": "Codeception Module for Slim framework.",
+    "keywords": [
+        "codeception",
+        "module",
+        "slim"
+    ],
+    "type": "library",
+    "license": "MIT",
+    "require": {
+        "php": "^7.2",
```

Comment:
You can make you library forward-compatible with upcoming PHP8 too: https://www.youtube.com/watch?v=c3bpTBjhK2Y

`php: >= 7.2`, see this PR https://github.com/symfony/symfony/pull/36876/files

Reply:
Until codeception and slim has restriction, it wouldn't matter too much. 
https://github.com/slimphp/Slim/blob/4.x/composer.json#L46
https://github.com/Codeception/Codeception/blob/4.1/composer.json#L18